banner.svg

```diff
@@ -0,0 +1,45 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="874" height="472" viewBox="0 0 874 472" xmlns:xlink="http://www.w3.org/1999/xlink">
```

Comment:
why do we need the svg banner?

Reply:
for Helm repository aggregator (https://artifacthub.io/) to which SoftwareMill Helm Repository will be added shortly - as you know, vector graphics (SVG is such) can scale with no loss, and having banner.png (raster) could have resulted in distortion/unreadable rendering. so just for quality purposes :)

Reply:
ok, but we should keep one. These banners change sometimes and maintaining two is unnecessary work